### ISSUE TITLE
Also cache files under css directory for slides.html5rocks.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#483]
```
What steps will reproduce the problem?
1. http://slides.html5rocks.com/#app-cache
2. turn off internet connection and refresh the page
3. the page only shows texts with no css style

What is the expected output? What do you see instead?
The pages shows as what we saw when network was on.

Please use labels and text to provide additional information.
Checked build_manifest.py, it only output all files under src/ to 
cache.appcache, but our css files are all under css/ dir. I think we need:
1) change the script to include css/ besides src/
2) replace all possible "\" in path to prevent the generated error when we run 
build_manifest.py in windows machine.
```

Original issue reported on code.google.com by `han...@google.com` on 8 Mar 2011 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0edb86a284.
```

Original comment by `ericbide...@html5rocks.com` on 8 Mar 2011 at 5:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, checked this in here: 
http://code.google.com/p/html5rocks/source/detail?r=0edb86a284
```

Original comment by `ericbide...@html5rocks.com` on 8 Mar 2011 at 5:55
